lib/onlyoffice_telegram_bugzilla_notifications.rb

```diff
@@ -31,32 +32,41 @@ def fetch_bugs_to_send
     end
 
     # Form message to send via telegram
-    def form_messages
+    # @param bugs_to_send [Array<Integer>] The array of bug IDs for which messages will be formed.
+    # @return [void] This method does not return a value.
+    def form_messages(bugs_to_send)
       @messages = []
-      @bugs_to_send.each do |bug|
+      bugs_to_send.each do |bug|
         @messages << Message.new(@bugzilla, bug)
       end
     end
 
-    # @param messages [Array<String>] message to send
-    def send_messages(messages = @messages)
-      Telegram::Bot::Client.run(@config['telegram_bot_token']) do |bot|
+    # Sends messages via Telegram.
+    # @param chat_config [Hash] The configuration hash for the chat, containing the bot token and channel ID.
+    # @param messages [Array<String>] The array of messages to send. Defaults to @messages.
+    # @return [void] This method does not return a value.
+    def send_messages(chat_config, messages = @messages)
+      Telegram::Bot::Client.run(chat_config['telegram_bot_token']) do |bot|
         messages.each_with_index do |message, index|
-          @logger.info("Sending info `#{message.for_logger} to chat #{@config['channel_id']}")
-          bot.api.sendMessage(chat_id: @config['channel_id'], text: message)
+          @logger.info("Sending info `#{message.for_logger} to chat #{chat_config['channel_id']}")
+          bot.api.sendMessage(chat_id: chat_config['channel_id'], text: message)
           @logger.info("Send info about bug: #{@bugs_to_send[index]}")
         end
       end
-      update_last_notified_bug(@bugs_to_send.last)
     end
 
     # Fetch info about not-notified bugs and send it
     def fetch_info_and_send
       fetch_bugs_to_send
-      return if @bugs_to_send.empty?
+      @config.each do |chat_name, chat_config|
+        next if chat_name == 'common_config'
```

Comment:
This is ugly :)
Maybe we can create some method that fill fetch only chat configurations?
Because current line implying that `common_config` one of the chat names, but we skip it